pkg/skaffold/deploy/kpt/kpt_test.go

```diff
@@ -1104,6 +1104,16 @@ func TestVersionCheck(t *testing.T) {
 				"local kustomize CLI to a version >= %v\nSee kustomize installation: %v",
 				kustomizeMinVersion, kustomizeDownloadLink),
 		},
+		{
```

Comment:
To fix the codedev blocker, suggest adding a sad path using the kustomize version value 
`{Version:unknown GitCommit:a0072a2cf92bf5399565e84c621e1e7c5c1f1094 BuildDate:2020-06-15T20:19:07+01:00 GoOs:darwin GoArch:amd64}`